### PR TITLE
adding dak related templates

### PR DESCRIPTION
**Pull request by ritikarawlani**

### CODE REVIEW

File: input/pagecontent/index-dak.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+<div>
+<p> This DAK and set of implementation tools are still undergoing development. </p>
+<p> Content is for demonstration purposes only. </p>
+</div>{:.stu-note}
+
+
+### Summary 
+This WHO <mark>[insert health domain here]</mark> DAK <mark>add content here</mark>
+
+### L1 Narrative Guidelines
```
Comment (natschja) [inline input/pagecontent/index-dak.md:11]: Can we align language, so L1 Narrative guidelines is OK (but sentence case)
```suggestion
### L1 Narrative guidelines
```

File: input/pagecontent/index-dak.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+ 
+- [Link to decision support logic]() 
+
+- [Link to scheduling logic]() 
+
+- [Link to indicators table]()
+ 
+- [Link to functional and non functional requirements]()
+
+### L3 FHIR Implementation Guide
```
Comment (natschja) [inline input/pagecontent/index-dak.md:30]:
```suggestion
### L3 Machine readable guidelines
```

File: input/pagecontent/index-dak.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+The L3 FHIR Implementation Guide for the <mark>[insert health domain here]</mark> SMART Guidelines is yet to be published. Links will be published here as soon as they're available.
+
+### L4 Reference Implementations
```
Comment (natschja) [inline input/pagecontent/index-dak.md:33]:
```suggestion
### L4 Executable guidelines
```

File: input/pagecontent/index-dak.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+Reference implementations representing the L4 layer for the <mark>[insert health domain here]</mark> SMART Guidelines are not yet available. Links will be published here as soon as they're available.
+
```
Comment (natschja) [inline input/pagecontent/index-dak.md:35]: Also please add: 

L5 Dynamic guidelines 
Content representing the L5 layer for the <mark>[insert health domain here]</mark> SMART Guidelines are not yet available. Links will be published here as soon as they're available.

File: _genonce.new.sh
Comment (costateixeira) [inline _genonce.new.sh:1]: this file is usually not needed - it is a leftover from the _updatepublisher and should have been automatically deleted.

**Pull request merged**